komga-webui/src/locales/en.json

```diff
@@ -164,12 +164,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
+    "edit_readlist": "Edit reading list"
   },
   "browse_series": {
-    "earliest_year_from_release_dates": "This is the earliest year from the release dates from all books in the series",
-    "series_no_summary": "This series has no summary, so we picked one for you!",
-    "summary_from_book": "Summary from book {number}:"
```

Comment:
Unless the summary is in the book somehow?
Still better to have "of", because it is a summary of the book (?)

Reply:
"This is the earliest year any book was released in the series" could probably be

"This is the earliest year of any released book in this series"

Reply:
"Picked summary for series without one."

I personally prefer the old one, but it could be expanded a little to be more helpful:
"This series has no summary, so one was randomly selected from one of the series books"